pkg/api/validation/validation.go

```diff
@@ -97,6 +97,10 @@ func ValidateAnnotations(annotations map[string]string, fldPath *field.Path) fie
 	if totalSize > (int64)(totalAnnotationSizeLimitB) {
 		allErrs = append(allErrs, field.TooLong(fldPath, "", totalAnnotationSizeLimitB))
 	}
+
+	if annotations[api.AffinityAnnotationKey] != "" {
```

Comment:
is there a difference between this and `if annotations[api.AffinityAnnotationKey] > 0` ?
Is one better than the other?